challenge.rb

```diff
@@ -1,26 +1,31 @@
-
 def capitalize_each_string(input)
-  #implement your solution here
+	#implement your solution here
+	input.map(&:capitalize)
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+	#implement your solution here	
+	input.select { |creature| creature == 'dog' }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+	#implement your solution here
+	input.find_all { |creature| creature != 'dog' }
```

Comment:
Ruby's `Enumerable` module has [a bunch of handy methods](http://ruby-doc.org/core-2.2.2/Enumerable.html). `#reject` is a pretty good fit for this one.